frontends/main/src/page-components/Header/Header.tsx

```diff
@@ -282,30 +280,19 @@ const Header: FunctionComponent = () => {
           <DesktopOnly>
             <StyledMITLogoLink logo="learn" />
             <LeftSpacer />
-            <MenuButton
-              text="Explore MIT"
-              onClick={toggler}
-              drawerOpen={drawerOpen}
-            />
+            <MenuButton text="Explore MIT" onClick={toggler} />
           </DesktopOnly>
           <MobileOnly>
-            <MenuButton onClick={toggler} drawerOpen={drawerOpen} />
+            <MenuButton onClick={toggler} />
             <LeftSpacer />
             <StyledMITLogoLink logo="learn" />
           </MobileOnly>
           <Spacer />
           <UserView />
         </StyledToolbar>
       </Bar>
-      <ClickAwayListener
```

Comment:
Removing the `ClickAwayListener` here broke the drawer on mobile:


https://github.com/user-attachments/assets/d045155f-8d9c-4246-b0a1-c10ab65ebbbf

I realize we created a close button in the drawer, but IMO clicking the menu button again should still close the drawer.

Reply:
Good catch. Should be resolved now.

Apparently ClickAwayListener attaches listeners to detect both click (mouse or touch) away and touchend (touch only).

Disabling the touch-only events fixes the issue, sort of. Click events also fire on touchscreens, but apparently 
- mousedown -> move lots -> mouse up = click event
- touchstart -> move lots -> touchend NOT trigger a click. (Movement must be fairly small.)

So maybe disabling the touch-only trigger would feel weird, so went with a different solution.